src/store/fs.rs

```diff
@@ -471,26 +428,6 @@ impl Actor {
         tx.send(tt).await.ok();
     }
 
-    async fn clear_dead_handles(&mut self) {
-        let mut to_remove = Vec::new();
-        for (hash, slot) in &self.handles {
-            if !slot.is_live().await {
-                to_remove.push(*hash);
-            }
-        }
-        for hash in to_remove {
-            if let Some(slot) = self.handles.remove(&hash) {
-                // do a quick check if the handle has become alive in the meantime, and reinsert it
```

Comment:
Getting rid of this nasty race condition handling was the whole purpose of the exercise. Also, we don't need a scheduled cleanup task anymore since the entity manager will deal with it. If an actor becomes idle it will be removed from the map after shutdown.